<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>425</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QListWidget" name="vaultList">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="5">
     <widget class="Line" name="line">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="addVaultButton">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>添加密码库</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" rowspan="3">
     <widget class="QWidget" name="vaultInfoWidget" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="13" column="0">
        <widget class="QWidget" name="passwordInputWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="5" column="0" colspan="2">
           <widget class="QProgressBar" name="progressBar">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="passwordLabel">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>请输入密码</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="progressReminder">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>两次输入的密码不相同!</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="passwordEdit">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="passwordTwiceLabel">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>请再次输入密码</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="passwordTwiceEdit">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="showPassword">
            <property name="text">
             <string>显示密码</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2" alignment="Qt::AlignLeft">
           <widget class="QPushButton" name="pushButton_2">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>解密</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="filePathLabel">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>解密文件夹路径：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="vaultPathReminderLabel">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(170, 0, 0)</string>
         </property>
         <property name="text">
          <string>密码库路径不存在！</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="vaultPathLabel">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>密码库路径：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="vaultNameLabel">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>密码库名称：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="5" rowspan="3">
     <widget class="QWidget" name="vaultConfigWidget" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolButton" name="vaultSettings">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>密码库设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="changePassword">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>修改密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="delVault">
         <property name="text">
          <string>删除密码库</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showPassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>update_password_echo_mode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vaultList</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateVaultInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delVault</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteVault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addVaultButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addVault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vaultSettings</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>editVaultSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>757</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_password_echo_mode(bool)</slot>
  <slot>launchVaultWin()</slot>
  <slot>enableVaultConfigWidget()</slot>
  <slot>updateVaultInfo()</slot>
  <slot>deleteVault()</slot>
  <slot>addVault()</slot>
  <slot>editVaultSettings()</slot>
 </slots>
</ui>
